apps/desktop/src/components/ProjectSettingsMenuAction.svelte

```diff
@@ -30,6 +30,11 @@
 		openExternalUrl(path);
 	});
 
+	shortcutService.on('open-in-finder', () => {
+		const path = `file://${project.path}`;
+		openExternalUrl(path);
+	});
```

Comment:
Should this be leveraging tauri's `revealItemInDir` method with their official `opener` plugin?

https://v2.tauri.app/reference/javascript/opener/#revealitemindir

```suggestion
	shortcutService.on('open-in-finder', async () => {
		try {
			// Use the Opener plugin to reveal the folder in the native file explorer
			await invoke('plugin:opener|reveal_item_in_dir', { path: project.path });
		} catch (e) {
			const message = `
				Failed to reveal directory in file explorer:

				${project.path}
			`;
			showToast({ title: 'Explorer error', message, style: 'error' });

			// Rethrowing for sentry and posthog
			throw e;
		}
	});
```

Reply:
Thanks for highlighting this!

It looks like the tuari opener function relies on the same `open` crate that our implementation already does. We seem to have extra handling for AppImages that the tauri plugin doesn't have.

In the longer term we should upstream our extra logic and move over to the tauri plugin, but for now I think it's fine.